sentry-contrib-native-sys/src/lib.rs

```diff
@@ -265,6 +310,50 @@ extern "C" {
     #[link_name = "sentry_uuid_as_string"]
     pub fn uuid_as_string(uuid: *const Uuid, str: *mut c_char);
 
+    /// Frees an envelope.
+    #[link_name = "sentry_envelope_free"]
+    pub fn envelope_free(envelope: *mut Envelope);
+
+    /// Serializes the envelope.
+    ///
+    /// The return value needs to be freed with `sentry_string_free()`.
+    #[link_name = "sentry_envelope_serialize"]
+    pub fn envelope_serialize(envelope: *const Envelope, size: &mut usize) -> *const c_char;
```

Comment:
```suggestion
    pub fn envelope_serialize(envelope: *const Envelope, size: *mut usize) -> *const c_char;
```

I know, just trying to keep this as straightforward a conversion as possible, dealing with anything more rust-like over in the other crate.